zipkin-lens/.gitignore

```diff
@@ -2,3 +2,5 @@
 *.log
 node_modules
 dist
+src/assets/scss/main.css 
```

Comment:
Can you not create main.css?

Reply:
It's autogenerated from the build process. But can check if there is a way

Reply:
Thank you!!